### ISSUE TITLE
[Screen Reader – Howdy – Scripts]: Table Summary and Captions are not defined for “Available Scripts” table in “Scripts” tab.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#13]
User Experience:
Users who rely on screen readers will affect to navigate, if Table Summary and Captions are not defined for “Available Scripts” table in “Scripts” tab.

Test Environment:
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://howdy.ai/
Browser: Edge
Screen Reader: Narrator

Repro Steps:
Step 1: Open the above URL in Edge browser.
Step 2: Open this URL: https://hcl-szz1925.howdy.ai/dashboard in edge.
Step 3: “Dashboard” page gets opened.
Step 4: Navigate to “Scripts” tab and select it.
Step 5: Navigate to “Available Scripts” table which is present in the page.
Step 5: Verify whether Table Summary and Captions are defined for “Available Scripts” table.

Actual Result:
Table Summary and Captions are not defined for “Available Scripts” table in “Scripts” tab.

Expected Result:
Table Summary and Captions should be defined for “Available Scripts” table in “Scripts” tab.

MAS Reference Link:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

![MAS1 3 1_Howdy_Table Summary](https://user-images.githubusercontent.com/48272005/54473420-68ad5a00-47fd-11e9-9a71-b8626a15213f.png)
[MAS1.3.1_Howdy_Table Summary.zip](https://github.com/howdyai/howdy-website/files/2973733/MAS1.3.1_Howdy_Table.Summary.zip)

**Comment by benbrown** [comment]
* Added a caption tag with a summary inline.

**Comment by v-gasikk** [comment]
Verified and issue is Fixed.
Now narrator is narrating the table summary and caption for the table.
